theano/gof/destroyhandler.py

```diff
@@ -1026,3 +1027,23 @@ def orderings(self, fgraph):
                         rval[app] = root_clients
 
         return rval
+
+
+class NoOutputFromInplace(Feature):
```

Comment:
I do not like the name, but do not have better suggestion. Also, I do not think this should be in this file. It should go in gof/toolbox.py where some other features are.